.changeset/seven-feet-beam.md

```diff
@@ -0,0 +1,6 @@
+---
+"@clerk/clerk-js": patch
+"@clerk/types": patch
+---
+
+Add `organizationCreateAvatarContainer` descriptor for logo upload box inside "Create Organization" flow
```

Comment:
Is this a good name? Any other suggestions?

Reply:
@LekoArts This component is also used in the `OrganizationProfile` not only on the `Create Organization` flow. So, I would name it like `organizationAvatarUploaderContainer`.